lasagne/layers/merge.py

```diff
@@ -68,22 +70,57 @@ class ElemwiseSumLayer(MergeLayer):
     before a :class:`DenseLayer`, insert separate :class:`DenseLayer` instances
     of the same number of output units and add them up afterwards. (This avoids
     the copy operations in concatenation, but splits up the dot product.)
```

Comment:
> But you might as well use other merge functions to aviod concatenation.

I can't think of any. And in any case, I think it's valuable to have the current note in the `ElemwiseSumLayer` documentation, because it explains an interesting and not-too-obvious use case for it. A more abstract / more general explanation for the `ElemwiseMergeLayer` would make it even less obvious.